content/en/docs/collector/deployment/gateway.md

```diff
@@ -218,3 +218,48 @@ Cons:
   https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor
 [spanmetrics-connector]:
   https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/spanmetricsconnector
+
+## Multiple Collectors / Single Writer Principle
+
+All metric data streams within
+OTLP must have a [single writer](/docs/specs/otel/metrics/data-model/#single-writer).
+When deploying multiple collectors in a gateway configuration, it's important to
+ensure that all metric data streams have a single writer and a globally unique
+identity.
+
+
+
+### Potential Problems
+
+Concurrent access from multiple applications that modify or report on
+the same data can lead to data loss or, at least, degraded data
+quality. An example would be something like inconsistent data from multiple sources
+on the same resource, where the different sources can overwrite each other because
+the resource is not uniquely identified.
```

Comment:
```suggestion
quality. For example, you might see inconsistent data from multiple sources
on the same resource, where the different sources can overwrite each other because
the resource is not uniquely identified.
```